Makefile

```diff
@@ -149,6 +149,14 @@ tc-image: container
 	@cp receptor packaging/tc-image/
 	@$(CONTAINERCMD) build packaging/tc-image -t receptor-tc
 
+receptorctl-test-venv: receptor
```

Comment:
Why does receptorctl-test-venv depend on receptor?

Reply:
this was juts a copy paste error